fe/fe-core/src/main/java/org/apache/doris/qe/ConnectProcessor.java

```diff
@@ -548,7 +561,21 @@ public TMasterOpResult proxyExecute(TMasterOpRequest request) {
                 UUID uuid = UUID.randomUUID();
                 queryId = new TUniqueId(uuid.getMostSignificantBits(), uuid.getLeastSignificantBits());
             }
-            executor.execute(queryId);
+            Context extractedContext = Telemetry.getOpenTelemetry().getPropagators().getTextMapPropagator()
+                    .extract(Context.current(), traceCarrier, getter);
+            try (Scope extract = extractedContext.makeCurrent()) {
```

Comment:
Here it should not be `extractedContext.makeCurrent()`, but `masterQuerySpan.makeCurrent()`.

Reply:
Done.